src/chapters/az-cli/walkthrough_az-cli.rst

```diff
@@ -0,0 +1,537 @@
+    
+    You can also view your defaults using the ``--list-defaults`` Argument:
+
+    .. sourcecode:: powershell
+
+        # -l can also be used as a shorthand
+        > az configure --list-defaults
+
+Resource Groups
+===============
+
+As always we will begin by provisioning a resource group to hold the other resources associated with it. Resource groups are an organizational tool to keep track of and manage resources as a whole instead of individually. One of the benefits of this approach is that it is easy to dispose of all of the resources by deleting the resource group they are contained in instead of deleting each of them one-by-one.
+
+Exploration
+-----------
+
+Let's begin by using our trusty help command to learn what management is available for resource groups. Although it's confusing remember that resource groups are accessed under the Group name ``group``.  
+
+.. sourcecode:: powershell
+
+    > az group -h
+
+Take a moment to look over the output. Because resource groups are simple containers there isn't much beyond the standard CRUD commands.
+
+Planning
+--------
+
+Of the available commands we can see that our goal is to ``create`` the resource. Let's dig into the ``create`` Command to see what Arguments are available.
+
+.. sourcecode:: powershell
+
+    > az group create -h
+
+We will need to provide at minimum the two **required** arguments:
+
+- ``-n`` or ``-g``: the name of the resource group
+- ``-l``: the Azure Location
+
+Fortunately we have already configured our default Location value of ``eastus`` so all we need to supply is the resource group name. We will use a consistent naming convention to make sure none of our resources have conflicting names. The convention for resource groups we will use is "your name" + "cli-wt" (CLI walkthrough) + "rg" (resource group):
+
+    <name>-cli-wt-rg
+
+Note that there is a 23 character limit for resource names. If you have a long first name consider using a short name like Pat if your name is Patrick.
+
+Provisioning
+------------
+
+Now that we have determined the command structure and its arguments we can create our resource group:
+
+.. sourcecode:: powershell
+
+    > az group create -n <name>-cli-wt-rg
+
+You should see a JSON output like this:
+
+.. sourcecode:: bash
+
+    {
+        "id": "/subscriptions/<subscription ID>/resourceGroups/<name>-cli-wt-rg",
+        "location": "eastus",
+        "managedBy": null,
+        "name": "<name>-cli-wt-rg",
+        "properties": {
+            "provisioningState": "Succeeded"
+        },
+        "tags": null,
+        "type": "Microsoft.Resources/resourceGroups"
+    }
+
+Notice how the subscription and location are set automatically. The former by logging in and the latter by configuring its default value.
+
+Configuring
+-----------
+
+Just as we set a default location we will assign this resource group as a default as well. Be sure to enter your new resource group name as the value:
```

Comment:
Suggested change from "Just as we set a default location we will assign this resource group as a default as well." to "Earlier we configured a default location, let's configure the default resource group too."